File: ETSMobile/ETSAboutViewController.m
//
//  ETSAboutViewController.m
//  ETSMobile
//
//  Created by Jean-Philippe Martin on 2014-04-06.
//  Copyright (c) 2014 ApplETS. All rights reserved.
//

#import "ETSAboutViewController.h"

@implementation ETSAboutViewController

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    
    [self.navigationController setNavigationBarHidden:NO animated:animated];
    [self.navigationController setToolbarHidden:YES animated:animated];
}

- (void)viewDidAppear:(BOOL)animated
{
    [super viewDidAppear:animated];
    
    [UIView animateWithDuration:0.8
                          delay:0
                        options: UIViewAnimationOptionCurveEaseInOut
                     animations:^{
                         self.aboutLabel.alpha = 1;
                     }
                     completion:nil];
}

@end

File: ETSMobile/ETSAboutViewController.swift
//
//  ETSAboutViewController.swift
//  ETSMobile
//
//  Created by Samuel Bellerose on 2015-11-11.
//  Copyright © 2015 ApplETS. All rights reserved.
//

import Crashlytics
import UIKit

class ETSAboutViewController: UIViewController{
    
    @IBOutlet weak var aboutTextLabel: UITextView!
    
    override func viewWillAppear(animated: Bool) {
        super.viewWillAppear(animated)
        
        Answers.logContentViewWithName("About ETSMobile", contentType: "About", contentId: "ETS-About", customAttributes: nil)
        
        self.navigationController?.setNavigationBarHidden(false, animated: animated)
        self.navigationController?.setToolbarHidden(true, animated: animated)
    }
}
